t/test.pl

```diff
@@ -313,7 +313,8 @@ sub _qq {
 # Trying to avoid setting $_, or relying on local $_ to work.
 sub display {
     my @result;
-    foreach my $x (@_) {
+    for (my $i = 0; $i < @_; $i++) {
+        my $x = $_[$i]; # Make a copy in case @_ contains unmodifiable elements
```

Comment:
Why not just
```perl
for my $arg (@_) {
    my $x = $arg;
```
? You're just taking a copy; no need to switch to a C-style for loop.

Reply:
I program more in C than Perl, so I have an unconscious bias towards that style, is all.  It's more perlish to do what you suggest.